Assets/Scripts/Tools/GraphArrangerTool.cs

```diff
@@ -17,6 +18,8 @@ public GraphArrangerTool(GraphArranger graphArranger, ToolPanelController toolPa
 
     public string ToolName => "Arrange";
 
+    private void SetToolgunModeText() => _toolPanelController.SetToolgunModeText($"{ToolName}\n{GetArrangerMode()}");
```

Comment:
Set sound to me like it should have an argument that is actually set. Consider changing name to RefreshToolgunModeText